### ISSUE TITLE
WebauthN and passwordless

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#2052]
Something that came in by email from Yao Ding at Facebook, 
<hr>
I’m representing the Consumer Deployment WG of FIDO Alliance to provide comments and seek clarification on 3.3.7 Accessible Authentication.

We have two main questions:

1. Clarify if providing WebAuthN is a sufficient technique even if in some cases the authenticator only supports PIN.
  **a.** Understanding 3.3.7 lists “Providing WebAuthN” as a sufficient technique.
  **b.** “Examples” language   –   “Common methods on laptops and phones are facial-scan, fingerprint, and pin number. The web site is not enforcing any particular use, it is assumed a user will setup a method that suits them.”  --  seems to suggest that PIN is always one in multiple options that the user can choose from.  We don’t think it’s accurate because when we deploy WebAuthN, WebAuthN does not provide any information on what modality an authenticator supports.  In some cases, the authenticator may only support PIN.
  **c.** As per definition of “cognitive function test”, memorizing a PIN is seemingly a cog test.
  **d.** Based on the above premise, will providing WebAuthN still pass 3.3.7 even in cases where the authenticator only supports PIN?
2. Clarify if passwordless authentication would pass 3.3.7
  **a.** Passwords are known to be weak for phishing and the industry is moving towards authentication without relying on passwords.
  **b.** Is it accurate to say that as long as the passwordless auth doesn’t rely on a cognitive function test, it passes 3.3.7?  
  **c.** If yes, may we provide examples of FIDO passwordless authentication to be included as examples or sufficient technques? 

Your response would be much appreciated!

Regards,
Yao

**Comment by alastc** [comment]
Proposed response: 

<hr>

> Clarify if providing WebAuthN is a sufficient technique even if in some cases the authenticator only supports PIN.

Essentially the SC is not trying to differentiate what is on the other side of webauthn, that is up to the user and their device / system.

The examples section is just that - examples of how it might work. 

> Based on the above premise, will providing WebAuthN still pass 3.3.7 even in cases where the authenticator only supports PIN?

If the user's device does not have anything except pin, then they must be happy or used to using a pin. If they cannot use a pin, then they wouldn't set-up a pin on that device for use with WebauthN.
From a web-authors point of view they are providing a potentially password-less (non-cog-fun-test) solution, but it is up to the user to have a device that supports that.

> Is it accurate to say that as long as the passwordless auth doesn’t rely on a cognitive function test, it passes 3.3.7?

Yes, and the last example is:
"A web site that requires 2-factor authentication allows for multiple options for the 2nd factor, including a USB-based method where the user simply presses a button to enter a time-based token."

> may we provide examples of FIDO passwordless authentication to be included as examples or sufficient techniques?

Would that be different from the last example? 

This would not be a place to advertise FIDO, but it would be appropriate to include a generic example if it is something _websites_ can use.

**Comment by yao-ding** [comment]
Thank you for the clarification @alastc !

I've updated the FIDO group with your response and the group had no follow up question.

If you don't have anything to add either, please feel free to close out the issue.
